### DISCUSSION TITLE
Is there a way to install EarTrumpet without going through the Microsoft Store App?

### DISCUSSION THREAD

**Discussion by Cr8zyIvan** [File-New-Project/EarTrumpet#1425] [Q&A] [answered]
Is there a downloadable and installable version of EarTrumpet? Something with a ".exe" file, or an auto-installer?

**Comment by riverar** [comment, marked as answer]
We try to stick with channels that support automatic upgrades. But we are also available via:
* [Windows Package Manager Client (winget)](https://github.com/microsoft/winget-cli) (`winget install File-New-Project.EarTrumpet`)
* Chocolatey (`choco install eartrumpet`)

Or you can just grab the bundle and extract it with Windows or 7-Zip (https://install.eartrumpet.app/master/EarTrumpet.Package_2.3.0.0_x86.appxbundle).

**Comment by Cr8zyIvan** [comment]
Gotcha. I tried to install EarTrumpet via the Package Manager Client. I see EarTrumpet seems to come in "App" form (not sure of the terminology) only. I was just wondering if EarTrumpet came in a more of a "Software" type of Format. With an installable exe File and all.

I'm used to more of an "old school" way of managing the Software I install on my Computers is all.

In any case, thank you for your reply. Cheers!

**Reply by riverar** [reply]
@Cr8zyIvan I hear ya. I do want to get loose files, packages, zips, etc. into the GitHub Releases feed. Would that be helpful?

**Reply by Cr8zyIvan** [reply]
Perhaps. But unless there's an installable exe file, or msi file, I don't think it would be that helpful (in my case at least).

Thank you very much for your repliy. It shows that you care about the software, and the people that use it.
